docs/content/homepage.md

```diff
@@ -6,7 +6,7 @@ slug: /
 hide_table_of_contents: true
 ---
 
-Welcome to Medusa, the open source Shopify alternative!
+Welcome to Medusa, the open-source Shopify alternative!
```

Comment:
We actually deliberately use "open source" so please change it back 🙂

Reply:
@shahednasser Done ✅ . Please have a look.